packages/accordion/src/vaadin-accordion-heading-styles.js

```diff
@@ -5,97 +5,24 @@
  */
 import '@vaadin/component-base/src/style-props.js';
 import { css } from 'lit';
-
-export const accordionHeading = css`
-  :host {
-    background: var(--vaadin-accordion-heading-background, transparent);
-    background-origin: border-box;
-    border: var(--vaadin-accordion-heading-border, none);
-    border-radius: var(--vaadin-accordion-heading-border-radius, var(--_vaadin-radius-m));
-    box-sizing: border-box;
-    color: var(--vaadin-accordion-heading-text-color, var(--_vaadin-color-strong));
-    display: block;
-    font-size: var(--vaadin-accordion-heading-font-size, inherit);
-    font-weight: var(--vaadin-accordion-heading-font-weight, 500);
-    gap: var(--vaadin-accordion-heading-gap, 0 var(--_vaadin-gap-container-inline));
-    height: var(--vaadin-accordion-heading-height, auto);
-    outline: calc(var(--vaadin-focus-ring-width) * var(--_focus-ring, 0)) solid var(--vaadin-focus-ring-color);
-    outline-offset: 1px;
-    padding: var(--vaadin-accordion-heading-padding, var(--_vaadin-padding-container));
-    -webkit-tap-highlight-color: transparent;
-    -webkit-user-select: none;
-    user-select: none;
-  }
-
-  :host([hidden]) {
-    display: none !important;
-  }
-
-  button {
-    align-items: center;
-    appearance: none;
-    background: transparent;
-    border: 0;
-    color: inherit;
-    display: flex;
-    font: inherit;
-    gap: inherit;
-    outline: none;
-    padding: 0;
-    touch-action: manipulation;
-  }
-
-  [part='toggle'] {
-    color: var(--_vaadin-color-subtle);
-  }
-
-  @media (prefers-reduced-motion: no-preference) {
-    [part='toggle'] {
-      transition-property: rotate;
-      transition-duration: 150ms;
-      animation: delay-initial-transition 1ms;
-    }
-
-    @keyframes delay-initial-transition {
-      0% {
-        rotate: 0deg;
-      }
-    }
-  }
-
-  [part='toggle']::before {
-    background: currentColor;
-    content: '';
-    display: block;
-    height: var(--vaadin-icon-size, 1lh);
-    mask-image: var(--_vaadin-icon-chevron-down);
-    width: var(--vaadin-icon-size, 1lh);
-    rotate: -90deg;
-  }
-
-  :host([disabled]) {
-    color: var(--_vaadin-color-subtle);
-  }
-
-  :host([disabled]) [part='toggle'] {
-    opacity: 0.5;
-  }
-
-  :host([dir='rtl']) [part='toggle']::before {
-    scale: -1;
-  }
-
-  :host([opened]) [part='toggle'] {
-    rotate: 90deg;
-  }
-
-  :host([dir='rtl'][opened]) [part='toggle'] {
-    rotate: -90deg;
-  }
-
-  @media (forced-colors: active) {
-    [part='toggle']::before {
-      background: CanvasText;
+import { summaryStyles } from '../../details/src/vaadin-lit-details-summary';
```

Comment:
Hmm, now I realize this creates a dependency to the details package. Is that a problem?

Reply:
It is already a dependency because of some shared mixin, so it's ok.